### ISSUE TITLE
Should we limit the number of Reset hint alerts after all atoms have broken away?

### ISSUE THREAD

**Issue by terracoda** [phetsims/friction#134]
In issue https://github.com/phetsims/friction/issues/119 @zepumph implemented the alerts perfectly.

New question for design meeting.

Are there too many "Reset" hint alerts after all atoms have broken away. Or is this repetition really useful to ensure users do in fact reset the sim.

The use case, I am slight concerned about is if a user, perhaps a teacher, is doing a repeat demo of Friction and all the atoms have broken away. After the evaporation point, the "Reset" hint will fire instead of the beyond evaporation point alert, e.g., "Jiggling super fast, super hot."

**Comment by terracoda** [comment]
I did a bit more testing with the simulation and I think it might be useful to use the same pattern as the  more explicit vertical alerts e.g., "At. top. Move down to rub harder." and  "Down. Rub fast or slow." (see https://github.com/phetsims/friction/issues/124).

@zepumph, is it possible to do something similar with the "Reset" alert?
For example, as it is doing now, but only twice, and then things go back to the appropriate content alerts, e.g., "Jiggling super fast, super hot."

So, if we look back at the number 8 and 9 alerts in issue https://github.com/phetsims/friction/issues/119, I'm thinking a better experience would be:

After two number 8 alerts, "Jiggling very fast, very hot." you get one "Reset" alert, and then again, after two more number 8 alerts, you get another "Reset" alert. Then you don't get any more until the sim is reset.

Oh, and I just noticed that the "Reset" is firing after the number 7 alert, rather than the number 8 alert. Ideally, it fires only after two of the number 8 alerts, "Jiggling super fast, super hot."

So ideally  the Reset would work like this:
- First single fire after two “Jiggling super fast, super hot.” alerts AND when there are no more atoms to break away. 
- Second single fire (again there has to be no more atoms to break away) and after two “Jiggling super fast, super hot.” alerts 
- Third time the condition is met, we just hear the regular number 8 alert, “Jiggling super fast, super hot.”

I think this might create an environment where the last thing a user hears before the alerts stop (see bug https://github.com/phetsims/friction/issues/131), will be something meaningful about the sim rather than an interaction hint.

@zepumph, I am going to assign back to you and un-assign @emily-phet for now.

Let me know if this is hard or if you have any questions.

**Comment by zepumph** [comment]
@terracoda I updated it like you asked. Please review. I also wanted to promote a TODO I found in the code to this issue.

> TODO: do we want a bit of a threshold hotter than the evap limit to hit to alert maximum?

Currently these alerts we are adjusting are triggered .0000001 (or so) above the evaporation temp. While you review, see if it would be nice if they were instead triggered closer to the visual "maximum" of the thermometer instead of at the evaporation limit.

I can also create another issue if we want to keep these separate.

**Comment by zepumph** [comment]
@terracoda and I talked in a meeting today, I'm going to create some space between evaporation limit and the max temp alert threshold.

@terracoda please review the implementation you expected to see from https://github.com/phetsims/friction/issues/134#issuecomment-428231849, as well as the max alert threshold (around the top of the thermometer or so.

**Comment by terracoda** [comment]
@zepumph, the new timing of the reset alerts sound good.
